cpp/non_iid_main.cpp

```diff
@@ -324,51 +320,42 @@ int main(int argc, char* argv[]) {
         if (t_tuple_res >= 0.0) {
             if (verbose == 1) printf("\tT-Tuple Test Estimate = %f / %d bit(s)\n", t_tuple_res, data.word_size);
             H_original = min(t_tuple_res, H_original);
+            tc635.t_tuple_res = t_tuple_res;
```

Comment:
Similarly, if both literal and bitstring assessments are conducted, only the literal assessment is saved.